### DISCUSSION TITLE
Cron <-> API Trigger Limitations?

### DISCUSSION THREAD

**Discussion by MentalGear** [Budibase/budibase#7677] [Help] [answered]
I'm starting to really really like Budibase, even more since I found out about the powerful and well designed build-in Automations.

I was wondering whether there are any rate limitations to the Cron Trigger when coupled to send out an REST API request (cloud hosted Budibase) ?
Specifically, I'd like to check an outside API endpoint every minute.

If this would work, it would obliterate the need for most outside Automation APIs (like Zapier or so) and greatly reduce complexity.

**Comment by melohagan** [comment, marked as answer]
Hey @MentalGear 

Glad you're enjoying Budibase!

The limitation on the Cron Trigger is once every minute, so you should be able to accomplish what you need!

**Reply by MentalGear** [reply]
Thx for the reply! Is this limit per cron trigger, or overall for all cron triggers a max. of 1 per minute on Budibase Cloud?

**Reply by melohagan** [reply]
The limit is per cron trigger, however in Budibase cloud all automations are queued and run one at a time. So it is conceivable that if the cloud environment was swamped with automations then it could cause some delays.
